mobile/src/components/shared/ICardPage.jsx

```diff
@@ -13,13 +14,15 @@ import {colors} from '../../utilites/Theme';
  
  
  
- 
+ //Adding experimental colors
 const statusColors = {
   active: colors.primary,
   inactive: colors.red,
   stale: colors.darkGray,
   'verifying account': colors.yellow,
   Unlinked: colors.lightGray,
+  'iNactive': colors.red,
+  'Inactive': colors.red
```

Comment:
Could we rename these two new states to more specific names. I think it would be very confusing to have 3 states called `inactive` with the only difference being capitalization?